EssentialCSharp.Web/wwwroot/js/site.js

```diff
@@ -152,7 +152,9 @@ const app = createApp({
         }
 
         function goToPrevious() {
-            window.location.href = "/" + PREVIOUS_PAGE;
+            if (!window.location.href.endsWith("/home")) {
+                window.location.href = "/" + PREVIOUS_PAGE;
+            }

```

Comment:
Is there a more generic solution we can think of? We have multiple pages (guidelines, etc.). Maybe use the sitemap to lookup if there is a match against a value in the sitemap or not?